kolibri/plugins/facility/assets/src/views/FacilityConfigPage/index.vue

```diff
@@ -23,7 +23,14 @@
       <div class="mb">
         <h2>{{ coreString('facilityLabel') }}</h2>
         <p class="current-facility-name">
-          {{ facilityName }}
+          {{ facilityName }} ({{ lastPartId }})
```

Comment:
We need to add a `$trs` message that combines `<facility name> (<id>)` because the RTL version might flip the two parts. Example `facilityNameWithId: '{facilityName} ({id})'`

Reply:
Great point, thanks. It's done now.